### PR TITLE
About me section

### PR DESCRIPTION
**Pull request by MMGGYY66**
adding about-me section and other files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,34 +34,188 @@ <h2>I am a Software Developer</h2>
...
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="button" type="button">See Project</button>
+                </article>
+            </section> <!-- end work section -->
+
+            <!-- start about me section -->
+            <section class="about-me">
```
Comment (Nyame-Wolf) [inline index.html:169]: I suggest you add the background image that is present in the figma design. 
This is a requirement for this project
Screenshot 2022-12-02 at 14 39 40
- This is your design
Screenshot 2022-12-02 at 15 59 23


- This is the figma design
Screenshot 2022-12-02 at 14 47 28

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @MMGGYY66  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:
- Professional readme
- Linters are passing
- Great use of semantic tags

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by TuyishimireEric** [comment]
# STATUS: INVALID CODE REVIEW REQUEST :stop_sign: :stop_sign:

## Hi @MMGGYY66  ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
### Invalid Code Review Request
**You submitted a PR which has failing Linters, :fearful:** 
Kindly try to fix it and make sure that all Linter result should be green, this means that the last commit have a green check mark (✅) or a summary of the passed checks is in the PR. 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

### Cheers and Happy coding! :clap::clap::clap:
---

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification._

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Invalid Code Review Request does not count into the code reviews limit.**

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @MMGGYY66 , 

I am Favour 👩🏽 , your reviewer for this project milestone.

Great Job implementing all the required changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Great Design and UI ✔️ 
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**